.github/workflows/quick.yaml

```diff
@@ -133,7 +133,7 @@ jobs:
         uses: actions/checkout@v4
 
       - name: Setup ccache
-        uses: squid-cache/ccache-action@v1.2.14
+        uses: hendrikmuhs/ccache-action@v1.2.14
```

Comment:
Using official ccache-action repository now works because I have adjusted Squid Project GitHub Actions [configuration](https://github.com/squid-cache/squid/settings/actions) to allow that action reuse.